code/datums/mind/mind.dm

```diff
@@ -570,6 +570,9 @@
 	mind.assigned_special_role = "Cultist"
 
 /datum/mind/proc/get_special_role_name()
```

Comment:
This can still return null, is that intended? Maybe you could make it `get_special_role_name(default = null)` to avoid having to pepper a dozen versions of `|| "No special role"`/`|| "unknown role"` all over the place.